explorer/lib/explorer_web/live/pages/home/index.html.heex

```diff
@@ -40,30 +40,44 @@
       <% end %>
     </.card>
     <.live_component module={ContractsComponent} id="contracts_card" class="sm:col-span-2" />
-    <%= if @latest_batches != :empty and @latest_batches != [] do %>
-      <.card
-        class="relative sm:col-span-2 w-full flex flex-col space-y-1"
-        inner_class="text-ellipsis overflow-hidden text-lg flex flex-col space-y-1"
-        title="Recent Batches"
-      >
+    <div class="relative truncate sm:col-span-2">
+        <h1 class="text-4xl sm:text-4xl font-bold font-foreground text-left py-2">
```

Comment:
Nice, I would only make the header smaller, try:
```suggestion
        <h1 class="text-xl font-bold font-foreground text-left py-2">
```

Reply:
addressed!